lib/bamboo/phoenix.ex

```diff
@@ -98,22 +97,57 @@ defmodule Bamboo.Phoenix do
     raise "function implemented for documentation only, please call: use Bamboo.Phoenix"
   end
 
-
   @doc """
   Sets the layout when rendering HTML templates
+
+  ## Example
+
+      def html_email_layout do
+        new_email
+        |> from("myapp@example.com")
+        # Will use MyApp.LayoutView with email.html layout when rendering
```

Comment:
For `text` and `html` layout functions, what do you think of doing `...email.html layout when rendering html emails